upup/pkg/fi/cloudup/gcetasks/instancegroupmanager.go

```diff
@@ -151,21 +151,15 @@ func (_ *InstanceGroupManager) RenderGCE(t *gce.GCEAPITarget, a, e, changes *Ins
 		}
 
 		if changes.TargetSize != nil {
-			request := &compute.InstanceGroupManagersResizeAdvancedRequest{
-				TargetSize: i.TargetSize,
-			}
-			if i.TargetSize == 0 {
-				request.ForceSendFields = append(request.ForceSendFields, "TargetSize")
-			}
-			op, err := t.Cloud.Compute().InstanceGroupManagers.ResizeAdvanced(t.Cloud.Project(), *e.Zone, i.Name, request).Do()
```

Comment:
Interesting!  The ForceSendFields was originally here because otherwise we didn't send Size == 0, but it looks like this has been fixed by using the Resize command.